solidity/contracts/bridge/Bridge.sol

```diff
@@ -121,6 +121,8 @@ contract Bridge is Governable, EcdsaWalletOwner {
 
     event MovingFundsBelowDustReported(bytes20 walletPubKeyHash);
 
+    event MovedFundsMerged(bytes20 walletPubKeyHash, bytes32 mergeTxHash);
```

Comment:
I like the consistency in keeping the sweeping term the two flows. It shouldn't matter that in the deposit case the sweep disables the refund, that's just a nice side effect - the main point of a sweep is that it consolidates the UTXOs to amortize the bitcoin fee and make it so that we don't have to build wallet software.

I went over all the names/tenses carefully and they all look good to me :+1:

Reply:
We can do it but I'd rename `movedFundsMergeRequests` to `movedFundsSweepRequests` not just `movingFunds`. This is because `movingFunds` will be ambiguous and it could also refer to the first step of the process which will be confusing. Also, the "moving" word is about moving from wallet to wallet so once the source wallet locks the funds on the target wallet, they are technically "moved" but just not swept yet. We also have the `MovedFundsMergeRequest` struct that must be renamed and `MovedFundsSweepRequest` looks much better than `MovingFunds` which is also the library name.

Reply:
After your explanation and looking further into the code `movedFundsSweepRequests` makes sense to me. I just realized this mapping is filled in `submitMovingFundsProof` so once the proof is accepted, funds are *moved* but not yet swept. I think it makes sense.

I also like we are separating moving vs moved states by naming of functions:
- `submitMovingFundsProof` - from Ethereum perspective they are still *moving*
- `movedFundsSweepRequests` - from Ethereum perspective they are *moved* but not yet swept.

Reply:
Here is the change around moved funds: https://github.com/keep-network/tbtc-v2/pull/244/commits/ecc7b50c6bdf6dc462f4295bf2b5bb4d95e35066 and here for disambiguating of deposit sweeps: https://github.com/keep-network/tbtc-v2/pull/244/commits/635e71a8fc0164531e12f75457fc4d52e794fd06. Note I have also renamed the `Sweep.sol` library to `DepositSweep.sol`.